package.json

```diff
@@ -75,12 +75,13 @@
     "ml-stat": "^1.3.3",
     "multiplet-analysis": "^2.0.0",
     "nmr-correlation": "^2.3.3",
-    "nmr-load-save": "^0.4.2",
+    "nmr-load-save": "0.4.2-pre.1666322459",
     "nmr-processing": "^9.2.0",
     "nmredata": "^0.9.0",
     "numeral": "^2.0.6",
     "openchemlib": "^8.0.1",
     "openchemlib-utils": "^2.0.0",
+    "playwright": "^1.27.1",
```

Comment:
?

Reply:
yes, it was a mistake, I forgot to re-install It globally, then it write the package.json

Reply:
you shouldn't need to install it globally. I never did it personally.

Reply:
In general, manually installing global npm packages is discouraged since `npx` exists.